structuredefinitions/Extension-England-BusinessChangeTypedDateTime.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-BusinessChangeTypedDateTime" />
+  <url value="https://fhir.nhs.uk/England/StructureDefinition/Extension-England-BusinessChangeTypedDateTime" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandBusinessChangeTypedDateTime" />
+  <title value="Extension England Business Change Typed Date Time" />
+  <status value="draft" />
+  <date value="2023-10-23" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This supports the exchange of dates and/or times.  The extension date and/or time is carried along with a code to identify the type of date and/or time." />
+  <purpose value="This extension extends the Encounter and the Communication resources to support the exchange of information describing the specific business related date and/or time." />
```

Comment:
The purpose talks about Encounter and Communication. However, in the contexts below, Organization and Organization Affiliation is stated. This seems to be inconsistent.